features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP_CellularNetwork.cpp

```diff
@@ -34,7 +34,15 @@ bool UBLOX_PPP_CellularNetwork::get_modem_stack_type(nsapi_ip_stack_t requested_
 
 AT_CellularNetwork::RegistrationMode UBLOX_PPP_CellularNetwork::has_registration(RegistrationType reg_type)
 {
+//<<<<<<< HEAD
```

Comment:
This is git edit here (can you clean up this file ( the commenst should not be added here)

Reply:
I pushed up this change.